### PR TITLE
Fix a syntax error and some spelling errors, improve typography

### PR DESCRIPTION
**Pull request by ariasuni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1400,7 +1400,7 @@ Okay, now it's deleted :confused:
 + `del x` doesn’t directly call `x.__del__()`.
 + Whenever `del x` is encountered, Python decrements the reference count for `x` by one, and `x.__del__()` when x’s reference count reaches zero.
 + In the second output snippet, `y.__del__()` was not called because the previous statement (`>>> y`) in the interactive interpreter created another reference to the same object, thus preventing the reference count to reach zero when `del y` was encountered.
-+ Calling `globals` caused the existing reference to be destroyed and hence we can see "Deleted!" being printed (finally!).
++ Calling `globals` caused the existing reference to be destroyed and hence we can see “Deleted!” being printed (finally!).
```
Comment (satwikkansal) [inline README.md]: I see a lot of quotes replaced (" with “) in the PR. Is there any difference in the meanings of both the characters. The one you replace it with isn't a character typed by standard keyboard.

Comment (ariasuni) [reply README.md]: They have the same meaning. These are the modern, better looking, more readable, English opening `“` and closing `”` quotes. There are easily available – though not widely known – on modern keyboard layouts.

If you don’t like it though, I can discard that change from my PR.

Comment (satwikkansal) [reply README.md]: > They have the same meaning. These are the modern, better looking, more readable, English opening “ and closing ” quotes. There are easily available – though not widely known – on modern keyboard layouts.

Interesting, I didn't knew that :)

> If you don’t like it though, I can discard that change from my PR.

The thing is, they are not ASCII. And given the fact anyone can contribute to the project, it'd be quite tedious to maintain consistency for quotes in the future examples. I hope you understand...

Comment (ariasuni) [reply README.md]: No problem, I totally understand. :)

**Comment by ariasuni** [comment]
I updated my branch :)

File: README.md [hunk 2]
```diff
@@ -2127,7 +2127,7 @@ Why did `Yo()._Yo__honey` worked? Only Indian readers would understand.
 
 ---
 
-## Section: Miscallaneous
+## Section: Miscellaneous
```
Comment (satwikkansal) [inline README.md]: You'll have to update the TOC as well :)

File: README.md [hunk 3]
```diff
@@ -2369,7 +2369,7 @@ If you have any wtfs, ideas or suggestions, please share.
 
 You can use these quick links for Twitter and Linkedin.
 
-[Twitter](https://twitter.com/intent/tweet?url=https://github.com/satwikkansal/wtfpython&hastags=python,wtfpython) | 
+[Twitter](https://twitter.com/intent/tweet?url=https://github.com/satwikkansal/wtfpython&hastags=python,wtfpython) |
 [Linkedin](https://www.linkedin.com/shareArticle?url=https://github.com/satwikkansal&title=What%20the%20f*ck%20Python!&summary=An%20interesting%20collection%20of%20subtle%20and%20tricky%20Python%20snippets.)
```
Comment (satwikkansal) [inline README.md]: That spa with the separator was intentional

**Comment by ariasuni** [comment]
Oops. This should be good now. I put ` | ` at the start of the next line so editing your document with editors that remove trailing lines will not break this again.

**Comment by satwikkansal** [comment]
And merged...

**Comment by satwikkansal** [comment]
Thank you so much @ariasuni  :smile:

**Pull request merged**